src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Avoid using literals in expressions (hardcoded values). Instead, define constants for the discount values and the minimum days required to qualify for those discounts. This will make the code more readable and maintainable.